### ISSUE TITLE
Full Page Reload

### ISSUE THREAD

**Issue by folamy** [hannoeru/vite-plugin-pages#112]
No matter what I edit in a route page, I get a full page reload every time, is this a known bug?

**Comment by folamy** [comment]
I can't provide a repo right now, but I have encountered this issue while using Vitesse Template sometimes ago. 
https://github.com/antfu/vitesse/issues/176#issue-981898277

This is a new project and this is what my code looks like:

```js
// router/index.ts
import generatedRoutes from 'virtual:generated-pages'
import { setupLayouts } from 'virtual:generated-layouts'
import { createRouter, createWebHistory, createMemoryHistory  } from 'vue-router'

const
  routes = setupLayouts(generatedRoutes),
  router = createRouter({
    history: import.meta.env.SSR ? createMemoryHistory() : createWebHistory(),
    routes
  })
```

in vite config:

```js
import Pages from 'vite-plugin-pages'
import Layouts from 'vite-plugin-vue-layouts'

export default defineConfig({
  plugins: [
    vue({
      template: { transformAssetUrls }
    }),
    Pages({
      // replaceSquareBrackets: true,
      extensions: ['vue', 'md'],
    }),
    Layouts()
  ] 
})
```
main.ts

```js
import { createApp } from 'vue'
import App from './App.vue'
import { router } from './router'

const app = createApp(App)

app.use(router)
app.mount('#app')
```
now if I try to edit any page in my '/page' dir, I get a full page reload on the browser

**Comment by hannoeru** [comment]
I can't repro this issue, the link to the vitesse issue is already fixed in v0.12.1.
